modoboa_installer/scripts/modoboa.py

```diff
@@ -228,6 +230,8 @@ def get_template_context(self):
             "radicale_enabled": (
                 "" if "modoboa-radicale" in extensions else "#"),
             "opendkim_user": self.config.get("opendkim", "user"),
+            "minute": random.randint(1, 59),
```

Comment:
minutes?

Reply:
run at the Xth minute ? idk, I changed it.